### ISSUE TITLE
Show HTML tag on TOC, not escaping them

### ISSUE THREAD

**Issue by leavesster** [facebook/docusaurus#1703]
## 🐛 Bug Report

TOC content doesn't show header as they show on H2/H3 tags.

As I mentioned in #1671 TOC content doesn't show HTML tags, TOC has escaped these HTML tags.
In that issue, I just said that localhost is different from the website. But my real problem is could the TOC allow HTML tag, not escape them ?

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

YES

## Expected behavior

![image](https://user-images.githubusercontent.com/17883920/61454227-83016b00-a98a-11e9-8b50-010379ddc9e1.png)

## Actual Behavior

![image](https://user-images.githubusercontent.com/17883920/61454196-65cc9c80-a98a-11e9-91c1-ef1d388b1a46.png)

## Reproducible Demo

(Paste the link to an example repo, including a `siteConfig.js`, and exact instructions to reproduce the issue.)

[fork-demo](https://github.com/leavesster/docusaurus/commit/89789b45f6d568b09ea5bb7fbfa5c60ade1f0e99)

### the way to fix it

and I find the way to fix it.
just change [code](https://github.com/facebook/docusaurus/blob/1edbeecbe565cbbbc0a6e6532c7c2d0c58d78807/packages/docusaurus-1.x/lib/core/nav/OnPageNav.js#L28)
from
`<Link hashLink={heading.hashLink} content={heading.content} />`
to
`<Link hashLink={heading.hashLink} content={heading.rawContent} />`

It seems the master branch is on developing v2.x, so I think maybe open a pull request to fix this is not a very good way.

I think this maybe also help for #1011 .
Thank you for creating this project ^_^.

**Comment by ThisIsMissEm** [comment]
This can now be closed as it's fixed @yangshun

**Comment by ThisIsMissEm** [comment]
Oh, wait, actually, no; I'm wrong there. @leavesster can you check with the current master of docusaurus as to whether the change in #1762 has the desired results for you?

**Comment by leavesster** [comment]
> Oh, wait, actually, no; I'm wrong there. @leavesster can you check with the current master of docusaurus as to whether the change in #1762 has the desired results for you?

I'm glad to try. In #1671 I said It only render wrong headers on my website, so I will produce it on my website.

I'm unfamiliar with dependencies with GitHub URL (not the whole git repo, only a directory in it).

I've read the npm document. And I could not find a way to add a URL (not git homepage, just https://github.com/facebook/docusaurus/tree/master/packages/docusaurus-1.x) to package.json ？

It just failed, and give me this message:
```bash
https://github.com/facebook/docusaurus/tree/master/packages/docusaurus-1.x: Extracting tar content of undefined failed, the file appears to be corrupt: "Invalid tar header. Maybe the tar is corrupted or it needs to be gunzipped?
```

I read [package.json](https://docs.npmjs.com/files/package.json) It seems
[urls-as-dependencies](https://docs.npmjs.com/files/package.json#urls-as-dependencies) only support tarball.
And [Git URLs as Dependencies](https://docs.npmjs.com/files/package.json#github-urls) only support the whole git, not a directory in git.

I want to know if you have any idea of testing unreleased but already on website package?

If there is no way I can use unreleased package. I think I can release a forked package tomorrow to check if it ok.

**Comment by yangshun** [comment]
@leavesster We just published v1.13.0, see if upgrading to that fixes your problems!

**Comment by leavesster** [comment]
> @leavesster We just published v1.13.0, see if upgrading to that fixes your problems!

It works. Thank you ^_^
